### PR TITLE
Move Workflows directory

### PR DESCRIPTION
**Pull request by Jack-Fraser16**
Troubleshooting why Workflows weren't integrated into PR check ... hopefully moving it out of the ISSUES_TEMPLATE directory and directly into the .githubs directory addresses this issue.

### CODE REVIEW

**Comment by jhlee-mitre** [comment]
Some dumb question. I see the msg "Some checks were not successful". That looks like workflow is integrated with PR. So  if I pull this code after merged, then whenever try to publish PR, I will see the same thing in the PR.

And it's okay the test is failed, or is it required to pass the test then be able to merge?

**Comment by Jack-Fraser16** [comment]
It

> Some dumb question. I see the msg "Some checks were not successful". That looks like workflow is integrated with PR. So if I pull this code after merged, then whenever try to publish PR, I will see the same thing in the PR.
> 
> And it's okay the test is failed, or is it required to pass the test then be able to merge?

Not a dumb question. It looks like the workflow is integrated with this PR; so yes, after this PR gets merged into main, the same check will be run on every PR. It it is my philosophy that any checks included as part of the PR process ought to be passed before merging in. Although I don't think it's an active requirement now, I think I can actually enforce that within the repo settings. For now, as long as a PR has an approval, it should be mergable.

**Review by jhlee-mitre — APPROVED** [review]
Approved.

**Pull request merged**
